cli/crates/typed-resolver-tests/tests/schema_types/interfaces.expected.ts

```diff
@@ -0,0 +1,22 @@
+export type Human = {
+  __typename?: 'Human'
+  id: string
+  firstName: string
+  lastName: string
+};
+
+export type Dog = {
+  __typename?: 'Dog'
+  id: string
+  name: string
+};
+
+export type LivingThing = {
+  metabolicRate: number
```

Comment:
Is this right? Should we have `__typename?: 'Dog' | 'Human'` here? It would be easy to implement. graphql-code-generator doesn't, because I assume they assume `__resolveType` as the mechanism for interfaces / unions typename.